packages/flame/lib/src/collisions/has_collision_detection.dart

```diff
@@ -4,10 +4,11 @@ import 'package:flame/game.dart';
 /// Keeps track of all the [ShapeHitbox]s in the component tree and initiates
 /// collision detection every tick.
 mixin HasCollisionDetection on FlameGame {
-  CollisionDetection<Hitbox> _collisionDetection = StandardCollisionDetection();
-  CollisionDetection<Hitbox> get collisionDetection => _collisionDetection;
+  CollisionDetection<ShapeHitbox> _collisionDetection =
+      StandardCollisionDetection();
+  CollisionDetection<ShapeHitbox> get collisionDetection => _collisionDetection;
 
-  set collisionDetection(CollisionDetection<Hitbox> cd) {
+  set collisionDetection(CollisionDetection<ShapeHitbox> cd) {
```

Comment:
Yeah I guess, a very minor one though since I doubt anyone has implemented their own collision detection system to pass in, but I can set the PR to breaking.